util/network.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util // import "istio.io/fortio/util"
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+)
+
+const (
+	// MinTCPPort is the minimum TCP port number that can be used by Fortio http server.
+	MinTCPPort int = 1024
+	// MaxTCPPort is the maximum TCP port number that can be used by Fortio http server.
+	MaxTCPPort int = 65535
```

Comment:
there must be such constant somewhere already in go ? but... more importantly:

also one can in theory use :svcname (from /etc/services) it doesn't have to be a number to work (current fortio uses a number for simplification but if we are support string it should just be passed to the listener as is)

Reply:
That's what I thought too. I checked the net pkg and was surprised to see no such constant.

Reply:
I thought it would be a good idea to validate the port number. I did not take into account using `/etc/services` to derive the port number. Let me see if there is an easy way to parse the file for the port. If not, I will remove port validation.

Reply:
don't parse the file, just use the user input and let listener tell you if it's valid/working or not